Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM continuumio/miniconda3
+
+# Set the working directory
+WORKDIR /app
+
+# Copy the env file
+COPY deploy/env.yml /app
+
+RUN conda env create -f /app/env.yml
+
+# Activate the new environment
+RUN echo "source activate myenv" > ~/.bashrc
+ENV PATH /opt/conda/envs/myenv/bin:$PATH
+
+COPY . /app
+WORKDIR /app/mlflow-learning
+
+ENV MLFLOW_TRACKING_URI=http://localhost:5008
+EXPOSE 5008
+
+
+CMD [ "/bin/sh", "-c", "mlflow server --backend-store-uri mlruns/ --default-artifact-root mlruns/ --host 0.0.0.0 --port 5008 \
```

Comment:
Can you move this _Dockerfile_ to `deploy/docker/`..
Also your `env.yml` must be inside `deploy/conda/`.
Refer [Project layout](https://docs.google.com/document/d/1FR--XjjbFCoMzxYAKFe5FJtTnjDpgRH9FFqMagEYl3g/edit)